templates/CRM/Member/Form/MembershipRenewal.tpl

```diff
@@ -226,8 +271,62 @@
     function adjustMembershipOrgType() {
       cj('#membershipOrgType').show();
       cj('#changeMembershipOrgType').hide();
+      cj('#showPriceSet').hide();
     }
 
+    {/literal}
+    {if $priceSet}
+      var priceSet = {$priceSet};
+    {else}
+      var priceSet = '';
+    {/if}
+      {literal}
+      var membershipTypeOrg = cj('#membership_type_id_0').val();
+      var membershipTypeId = cj('#membership_type_id_1').val();
+
+      function showPriceSet() {
+      {/literal}
+        {if $price_set_has_new_orgs}{literal}
```

Comment:
`showPriceSet` still remains fairly specific to the membership renewal.  We *might* be able to generalize it, but I do think this might border coincidental abstraction and in the long term increase to brittleness.